lib/3scale_toolbox/commands/import_command/import_csv.rb

```diff
@@ -0,0 +1,176 @@
+require 'cri'
+require 'uri'
+require 'csv'
+require '3scale/api'
+require '3scale_toolbox/base_command'
```

Comment:
shouldn't both `3scale/api` and `3scale_toolbox/base_command` be assumed to have been required by the command  loader?

Reply:
I'm definitely for not assuming anything. At some point you are going to test this in isolation and it will fire back.